src/views/DataExploration/components/tabs/Biospecimens/index.tsx

```diff
@@ -150,6 +155,12 @@ const BioSpecimenTab = ({
               },
             })
           ),
+        extra: [
+          <Button onClick={(e) => downloadReport(ReportType.BIOSEPCIMEN_DATA)}>
```

Comment:
Checker si on veut un button type="primary"

Reply:
according the the figma done by Lucas. No its not a primary.